FSiAdminSecurityBundle.php

```diff
@@ -43,6 +45,22 @@ public function build(ContainerBuilder $container)
         }
     }
 
+    public function boot()
+    {
+        /* @var $doctrine RegistryInterface */
+        $doctrine = $this->container->get('doctrine');
+        $userClass = $this->container->getParameter('admin_security.model.user');
+        $userRepositoryClass = 'FSi\Bundle\AdminSecurityBundle\Security\User\UserRepositoryInterface';
+        $repository = $doctrine->getRepository($userClass);
```

Comment:
I think this operation should be faster (without actually instantiating the repository):
`$repositoryClass = $doctrine->getManagerForClass($userClass)->getClassMetadata($userClass)->customRepositoryClassName`